eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -839,6 +840,41 @@ mod tests {
         assert_eq!(finalized_slot, finalized_slot_1);
     }
 
+    #[test]
+    fn test_finality_light_client_update_correctness() {
+        const TREE_FINALITY_DEPTH: usize = 6;
+        const TREE_FINALITY_INDEX: usize = 41;
+        const TREE_NEXT_SYNC_COMMITTEE_DEPTH: usize = 5;
+        const TREE_NEXT_SYNC_COMMITTEE_INDEX: usize = 23;
+
+        let config_for_test = get_test_config();
+
+        let relay = get_relay(true, true, &config_for_test);
+
+        let light_client_update = relay.beacon_rpc_client.get_finality_light_client_update_with_sync_commity_update().unwrap();
+
+        let branch: Vec<ethereum_types::H256> = light_client_update.finality_update.finality_branch.iter().map(|h| h.0).collect();
+        assert!(merkle_proof::verify_merkle_proof(
+            light_client_update.finality_update
+                .header_update
+                .beacon_header
+                .tree_hash_root(),
+            branch.as_slice(),
+            TREE_FINALITY_DEPTH,
+            TREE_FINALITY_INDEX,
+            light_client_update.attested_beacon_header.state_root.0
+        ));
+
+        let branch = light_client_update.sync_committee_update.as_ref().unwrap().next_sync_committee_branch.iter().map(|h| h.0).collect::<Vec<ethereum_types::H256>>();
+        assert!(merkle_proof::verify_merkle_proof(
+            light_client_update.sync_committee_update.as_ref().unwrap().next_sync_committee.tree_hash_root(),
+            branch.as_slice(),
+            TREE_NEXT_SYNC_COMMITTEE_DEPTH,
+            TREE_NEXT_SYNC_COMMITTEE_INDEX,
+            light_client_update.finality_update.header_update.beacon_header.state_root.0
+        ));
```

Comment:
Please add messages to `assert` statements. This way, it's better for debugging and also improves readability.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/837/commits/8cc5447c45673c2c6d0a6094172103918482262a